src/features/timeline/TimelineIndexPage.js

```diff
@@ -44,22 +42,30 @@ const CycleIndexPage = ({ entryId }) => {
     })
   }, [selectedDate])
 
+  const notHasPlacedPeriod =
+    !entries.length || !entries.filter((s) => s.tags.length).length
+
+  useEffect(() => {
+    if (notHasPlacedPeriod) {
+      navigate("/incomplete")
```

Comment:
**nitpick:** Add this as a navigation constant in `navigation/navItems` similar to line 87 `TIMELINE`

You may skip `primary` and `icon` for this one as not needed.

Reply:
All updated with constants :)